models/classes/runner/SessionCsrfToken.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
+ */
+
+namespace oat\taoTests\models\runner;
+
+/**
+ * Class SessionCsrfToken
+ *
+ * Handles CSRF token through the PHP session
+ *
+ * @package oat\taoQtiTest\models\runner
+ */
+class SessionCsrfToken implements CsrfToken
```

Comment:
Is there a way to test that class ? The difficulty is to mock `PHPSession::singleton();` but if you could add a test, it would rocks !